app/models/concerns/users/presentable.rb

```diff
@@ -20,8 +20,8 @@ def hashed_email
       Digest::MD5.hexdigest(email.downcase)
     end
 
-    def gravatar_url(d = "identicon")
-      "https://www.gravatar.com/avatar/#{hashed_email}?s=300&d=#{d}"
+    def gravatar_url(d = "identicon", s = "300")
```

Comment:
Did you use this change anywhere? If not I'd argue we revert it but up to you.

Reply:
I was going to use it but I didn't. This change does allow for better sizing though so there's no reason not to keep it imho.